pykafka/balancedconsumer.py

```diff
@@ -398,6 +400,10 @@ def _set_watches(self):
             self._zookeeper, self._consumer_id_path,
             self._consumers_changed
         )
+
+        self._zookeeper.add_listener(self._zookeeper_state_changed)
+        self._zookeeper_connected = self._zookeeper.state == KazooState.CONNECTED
```

Comment:
Ah right, thanks, I see the point now. So then, wouldn't it be ok to just do
```
self._zookeeper_connected = self._zookeeper.state
```
so that we don't risk clobbering the KazooClient's internal state by writing CONNECTED ourselves, and still make sure we initialise `self._zookeeper_connected` correctly?